ga4gh/backend.py

```diff
@@ -33,6 +33,7 @@ def __init__(self):
         self._callSetIds = []
         self._requestValidation = False
         self._responseValidation = False
+        self._defaultPageSize = 100
```

Comment:
It is - see the DEFAULT_PAGE_SIZE key. This is just to give it an initial value. We could set all these to None I suppose, but what if we want to test this class in isolation?